docs/integrations/databases/open-telemetry/redis.md

```diff
@@ -0,0 +1,135 @@
+  *   Save the `redis.conf` file and restart the redis server using command : `sudo  service redis-server restart`
+
+Once the logs are configured to be written to a local file, follow the below step to configure collection in Sumo.
+
+##Collection Configuration & App installation
+
+As part of setting up the collection process and app installation user can select the App from App Catalog and click Install App. Please follow the steps below:
+
+### Step1: Set up Collector
+
+If you want to use an existing OpenTelemetry Collector, then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector, please select the "Add a new Collector" option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command that can be executed in the machine that needs monitoring. Once executed, it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/collector-screenshot.png')} alt="Collector" />
+
+###Step2: Configure integration
+
+Open Telemetry works with a [configuration](https://opentelemetry.io/docs/collector/configuration/) yaml file with all the details concerning the data that needs to be collected. For example, it specifies the location of a log file that is read and sent to Sumo Logic platform.
+
+In this step we will be configuring the yaml required for Redis Collection.
+
+The log file path configured to capture redis logs must be given here.
+
+The files are typically located in `/var/log/redis/redis-server.log`. If you are using a customized path, check the `[redis.conf](https://download.redis.io/redis-stable/redis.conf)` file for this information.
+
+Click on the "Download YAML File" button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/configuration-screenshot.png')} alt="Configuration" />
+
+###Step3: Sending logs to Sumo
+
+Once you have the yaml file downloaded in step 2, please follow the below steps based on your environment
+
+####Linux
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command `sudo systemctl restart otelcol-sumo`
+
+####Mac
+
+1.  Copy the yaml at `/etc/otelcol-sumo/conf.d/` folder in the Redis instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the command `otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --conf "glob:/etc/otelcol-sumo/conf.d/*.yaml"`
+
+After the successful execution of the above command, Sumo will start receiving the data from your host machine.
```

Comment:
I'd rephrase this as:
Sumo Logic will start receiving data from your host machine when the above command is successfully executed.